packages/perseus/src/widgets/input-number/input-number.stories.tsx

```diff
@@ -108,3 +109,23 @@ export const Percent = (args: InputNumberOptions): React.ReactElement => {
     );
 };
 Percent.args = question3.widgets["input-number 1"].options;
+
+export const Answerful = (): React.ReactElement => {
+    const item = getAnswerfulItem("input-number", {
+        simplify: "optional",
+        size: "normal",
+        value: 42,
+    });
+    item.question.content = `The answer is 42\n${item.question.content}`;
+    return <ServerItemRendererWithDebugUI item={item} />;
+};
+
+export const Answerless = (): React.ReactElement => {
+    const item = getAnswerfulItem("input-number", {
+        simplify: "optional",
+        size: "normal",
+        value: 42,
+    });
+    item.question.content = `The answer is 42\n${item.question.content}`;
+    return <ServerItemRendererWithDebugUI item={item} startAnswerless />;
+};
```

Comment:
I think Jeremy must have skipped this widget for converting the stories to the new CSF format. Would be nice if these stories were in that format as well as we're trying to switch them all over to it, but looks like you'd have to convert the older stories for that to work, so totally up to you.

Reply:
I talked to him and I guess there were reasons for not doing InputNumber